### ISSUE TITLE
Crash preventing launch again now on 1.21.1

### ISSUE THREAD

**Issue by twenty41** [Globox1997/RecipeRemover#8]
### Minecraft Version

1.21.1

### Mod Loader Version

0.16.5

### Mod Version

1.0.9

### Describe the Issue

Crash before launch after install
```
[21:35:42] [main/ERROR]: Mixin apply for mod reciperemover failed reciperemover.mixins.json:ServerRecipeBookMixin from mod reciperemover -> net.minecraft.class_3441: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @WrapOperation annotation on errorMixin could not find any targets matching 'handleList' in net/minecraft/class_3441. Using refmap reciperemover-refmap.json [INJECT_PREPARE Applicator Phase -> reciperemover.mixins.json:ServerRecipeBookMixin from mod reciperemover -> Prepare Injections -> wrapOperation$hac000$reciperemover$errorMixin(Lorg/slf4j/Logger;Ljava/lang/String;Ljava/lang/Object;Lcom/llamalad7/mixinextras/injector/wrapoperation/Operation;)V -> Parse ->  -> Validate Targets]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @WrapOperation annotation on errorMixin could not find any targets matching 'handleList' in net/minecraft/class_3441. Using refmap reciperemover-refmap.json [INJECT_PREPARE Applicator Phase -> reciperemover.mixins.json:ServerRecipeBookMixin from mod reciperemover -> Prepare Injections -> wrapOperation$hac000$reciperemover$errorMixin(Lorg/slf4j/Logger;Ljava/lang/String;Ljava/lang/Object;Lcom/llamalad7/mixinextras/injector/wrapoperation/Operation;)V -> Parse ->  -> Validate Targets]
	at org.spongepowered.asm.mixin.injection.selectors.TargetSelectors.validate(TargetSelectors.java:346) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:369) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:340) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at com.llamalad7.mixinextras.injector.MixinExtrasInjectionInfo.<init>(MixinExtrasInjectionInfo.java:14) ~[mixinextras-0.4.1-215bdc40eef77b8f.jar:?]
	at com.llamalad7.mixinextras.injector.MixinExtrasLateInjectionInfo.<init>(MixinExtrasLateInjectionInfo.java:19) ~[mixinextras-0.4.1-215bdc40eef77b8f.jar:?]
	at com.llamalad7.mixinextras.injector.wrapoperation.WrapOperationInjectionInfo.<init>(WrapOperationInjectionInfo.java:28) ~[mixinextras-0.4.1-215bdc40eef77b8f.jar:?]
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:196) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:664) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1399) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:731) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:315) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:246) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:437) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:418) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.15.3+mixin.0.8.7.jar:0.15.3+mixin.0.8.7]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.16.5.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at net.minecraft.class_7378.<clinit>(class_7378.java:44) ~[client-intermediary.jar:?]
	at net.minecraft.class_7923.method_47485(class_7923.java:240) ~[client-intermediary.jar:?]
	at net.minecraft.class_7923.method_47482(class_7923.java:255) ~[client-intermediary.jar:?]
	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:986) [?:?]
	at net.minecraft.class_7923.method_47487(class_7923.java:253) [client-intermediary.jar:?]
	at net.minecraft.class_2966.redirect$coi000$fabric-registry-sync-v0$initialize(class_2966.java:565) [client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:65) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:130) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.5.jar:?]
```

### latest.log

[latest.log](https://github.com/user-attachments/files/17068662/latest.log)

### Do you use any other mods except the required ones?

Yes but still happened on isolated instance

**Comment by CheddarBeCheese** [comment]
im having this same problem in 1.21

**Comment by Globox1997** [comment]
Are you using the latest version of this mod?

**Comment by Globox1997** [comment]
Pretty sure this is fixed in 1.0.10, reopen if not.
